packages/massa-proto-cli/src/protobuf.ts

```diff
@@ -71,9 +71,15 @@ export async function getProtoFunction(protoPath: string): Promise<ProtoFile> {
       type: (field as { type: string; id: number }).type,
     }));
   const rHelper = protoJSON.nested[messageNames[1]] as IType;
-  const rHelperKeys = Object.keys(rHelper.fields);
-  const resType =
-    rHelperKeys.length === 1 ? rHelper.fields[rHelperKeys[0]].type : 'void';
+  let resType: string;
+  // if the rHelper.fields exists, get the return type
+  try {
+    const rHelperKeys = Object.keys(rHelper.fields);
+    resType =
+      rHelperKeys.length === 1 ? rHelper.fields[rHelperKeys[0]].type : 'void';
+  } catch (e) {
+    resType = 'void';
+  }
 
```

Comment:
Is it possible to use a condition instead of try catch

```
let resType = 'void';
if (rHelper && rHelper.fields) {
  const rHelperKeys = Object.keys(rHelper.fields);
  resType = rHelperKeys.length === 1 ? rHelper.fields[rHelperKeys[0]].type : 'void';
}
```

Something like that

Reply:
i already updated this part in the second pr https://github.com/massalabs/massa-sc-toolkit/pull/327